coordinator/pom.xml

```diff
@@ -734,6 +734,11 @@
         <artifactId>logback-core</artifactId>
         <version>${logback.version}</version>
       </dependency>
+      <dependency>
+        <groupId>ch.qos.logback</groupId>
+        <artifactId>logback-access</artifactId>
+        <version>${logback.version}</version>
+      </dependency>
```

Comment:
Noticed from IDEA there was a dependency to older version of `logback-access` than `logback-core`. And logs had:

    2025-04-15T16:40:08.8618576Z INFO  [Exec Stream Pumper] 2025-04-15 16:40:08,792 ProcessRunner.java:78 - stargate1-0> Caused by: java.lang.ClassNotFoundException: ch.qos.logback.core.hook.DelayingShutdownHook not found by io.stargate.db.dse_next [1]

which could be due to old(er) version of `logback-core`. Not sure this will fix it, but will try.

Reply:
Does not change things; ClassNotFoundException remains.

Reply:
Not really sure where logback 1.2.x comes through: it could be due to dropwizard-metrics 4.2.18 (not latest, but all 4.2.x have same dep) which depends on 1.2.11... but we have managed-dependency force version so really unclear.
Even jars under `coordinator/stargate-lib` have 1.3.15 version.